extensions/notebook/src/jupyter/jupyterSessionManager.ts

```diff
@@ -333,6 +333,10 @@ export class JupyterSession implements nb.ISession {
 	private async setEnvironmentVars(skip: boolean = false): Promise<void> {
 		if (!skip && this.sessionImpl) {
 			let allCode: string = '';
+			// Ensure cwd matches notebook path (this follows Jupyter behavior)
```

Comment:
how do we check this doesn't break any existing scenario? do we have test for it? if not should we add a task to add test later?

Reply:
We don't 😞. #8161 to track